CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.0.2] - 2019-02-23
```

Comment:
It should be 2.1.0 according to SemVer (as new interface was added). I know this is "fix" from JSON-API point of view, but SemVer is from "code" point of view. 

https://semver.org/
> A bug fix is defined as an internal change that fixes incorrect behavior.

And while new interface can fix incorrect behavior, it is not "internal" change.